pkg/network/netlink/conntracker.go

```diff
@@ -174,40 +189,19 @@ func (ctr *realConntracker) GetTranslationForConn(c network.ConnectionStats) *ne
 	return t.IPTranslation
 }
 
-func (ctr *realConntracker) GetStats() map[string]int64 {
-	// only a few stats are locked
-	ctr.RLock()
-	size := ctr.cache.cache.Len()
-	orphanSize := ctr.cache.orphans.Len()
-	ctr.RUnlock()
-
-	m := map[string]int64{
-		"state_size":  int64(size),
-		"orphan_size": int64(orphanSize),
-	}
-
-	gets := ctr.stats.gets.Load()
-	getTimeTotal := ctr.stats.getTimeTotal.Load()
-	m["gets_total"] = gets
-	if gets != 0 {
-		m["nanoseconds_per_get"] = getTimeTotal / gets
+func (ctr *realConntracker) RefreshTelemetry() {
+	for {
+		ctr.RLock()
+		stateSize.Set(float64(ctr.cache.cache.Len()))
+		orphanSize.Set(float64(ctr.cache.orphans.Len()))
+		ctr.RUnlock()
+		time.Sleep(time.Duration(5) * time.Second)
 	}
+}
 
-	registers := ctr.stats.registers.Load()
-	m["registers_total"] = registers
-	registersTotalTime := ctr.stats.registersTotalTime.Load()
-	if registers != 0 {
-		m["nanoseconds_per_register"] = registersTotalTime / registers
-	}
-
-	unregisters := ctr.stats.unregisters.Load()
-	unregisterTotalTime := ctr.stats.unregistersTotalTime.Load()
-	m["unregisters_total"] = unregisters
-	if unregisters != 0 {
-		m["nanoseconds_per_unregister"] = unregisterTotalTime / unregisters
-	}
-	m["evicts_total"] = ctr.stats.evicts.Load()
-
+func (ctr *realConntracker) GetStats() map[string]int64 {
```

Comment:
Do we need the `GetStats` endpoints anymore? We should only keep the stats necessary to service the agent status command.

Reply:
I am preserving the functionality of this block: https://github.com/DataDog/datadog-agent/blob/main/pkg/network/tracer/tracer.go#L483-L517

Reply:
I thought we were getting rid of most of the stats in that block?

Reply:
latest commit removes most of the getStats calls except for state delta calculation or agent status